docs/PID tuning.md

```diff
@@ -4,30 +4,70 @@ Every aspect of flight dynamics is controlled by the selected "PID controller".
-you're commanding with your sticks. An error is computed which is the difference between your target rotation rate and
-the actual one measured by the gyroscopes, and the controller tries to bring this error to zero.
+For example:
+* You push the pitch stick forward 10%.
+* Cleanflight determines that, based on your rates and expo, this corresponds to an angular rate of 50 degrees per second.
+* The PID controller measures the current angular rate being reported by the gyros. Say that it is 0 degrees per second.
+* The error is now 50 - 0 = 50 degrees per second.
+* The PID controller adjusts motor output to increase the angular rate so as to reduce the error.
+
+These calculations are simultaneously happening on all three axes of control: pitch, roll, and yaw. Motor output is constantly adjusted to make the rate of rotation try to match what is being commanded by the sticks.
 
 ##PIDs
 
 **The P term** controls the strength of the correction that is applied to bring the craft toward the target angle or
-rotation rate. If the P term is too low, the craft will be difficult to control as it won't respond quickly enough to
-keep itself stable. If it is set too high, the craft will rapidly oscillate/shake as it continually overshoots its
-target.
+rotation rate. The larger the error, the larger the P term will be. When a driver is driving the car, and the car just barely starts to drift out of the lane (small error) the driver makes a small correction to get the car back in the lane. If the driver were to look up and the car was way out of the lane and going into the ditch (large error) the driver might make a much larger correction. The strength of the correction is proportional to the magnitude of the error. This is how the P term works.
+
+If the P term is too low, the craft will be difficult to control as it won't respond strongly enough to
+keep itself stable. A craft with low P gain might feel as if it was "drunk". The pilot might feel as if they were constantly one step behind the quad--the pilot is responding to the quad, instead of the other way around.
+
+If P is set too high, the craft will rapidly oscillate/shake as it continually overshoots its target. Excess P gain is the single easiest thing to troubleshoot in multirotor tuning because the oscillations that result are so distinct. Excess P gain is demonstrated in [this video](https://www.youtube.com/watch?v=RMQXSB5-6bE). 
+
+A good guideline for setting P gain is to increase P until oscillations start to occur during normal control input (e.g. moderate turns and forward flight). Then decrease P to between 50% and 80% of that value. Another way of determining the oscillation threshold is to riase the throttle quickly from a hover and look/listen for oscillations during the punchout. Often times, espeically if flying line-of-sight, oscillations will be easier to hear than to see.
```

Comment:
Higher P gains are not the primary way of setting responsiveness to control inputs.  RCrate and PitchRoll rate have a much wider range of usable values and a bigger influence on stick responsiveness.  P gains are  vital for keeping the quad stable, i.e. the ability to resist external forces like gusts of wind.  

Maybe change that line to:   "Higher P gain helps the craft better resist external forces, such as gusts of wind, from pushing it off its intended angle in flight, and tends to give sharper responses to control inputs."  Then next sentence.. 

"One of the most common *mistakes* made by novice tuners is having too much P"  

Maybe then

"Often this causes visible wobbles and oscillations after sudden stick inputs."  

then "Even if these oscillations.... "